### PR TITLE
updated the seminar and bio for Pete and Mozhgan

### PR DESCRIPTION
**Pull request by mozhgan-kch**

### CODE REVIEW

File: site/files/seminar_files/bio/david_hubber.md [hunk 1]
```diff
@@ -0,0 +1 @@
+David Hubber is an astrophysicist by training that has worked extensively on writing software such as GANDALF and SEREN to simulate star forming regions. He has been a researcher at the university of Cardiff, the university of Sheffield, and is currently a Postdoc researcher Ludwig-Maximilians-Universitat Munich.
```
Comment (willfurnass) [inline site/files/seminar_files/bio/david_hubber.md:1]: Links to GANDALF and SEREN?  Links to his own website / profile page / twitter?

File: site/files/seminar_files/bio/joz_martin.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Jos Martin is the senior engineering manager for parallel computing products at MathWorks. He has responsibility for all parts of Parallel Computing Toolbox, including the use of GPU and big data types in other areas. In addition, he is also responsible for MATLAB Drive, MATLAB Connector, and products that help link the desktop to the cloud. Before moving to a development role within MathWorks, he worked as a consultant in the U.K., writing large-scale MATLAB applications, particularly in the finance and automotive sectors. Prior to joining MathWorks in 2000, he held a Royal Society Post-Doctoral Fellowship at the University of Otago, New Zealand. His area of research was Experimental Bose-Einstein Condensation (BEC), a branch of low-temperature atomic physics.
```
Comment (willfurnass) [inline site/files/seminar_files/bio/joz_martin.md:1]: NB typo in Markdown file name or in the name in this para?  Links to MATLAB products he's responsible for?

File: site/pages/community/seminars.md [hunk 3]
```diff
@@ -13,18 +13,17 @@ If you would like to recommend a speaker or would like to give a talk then pleas
 
 ### September 12, 2018 &mdash; 15:30-16:30, Hicks building, lecture theatre 7
 <strong>Rob Baxter (EPCC)</strong><br/><br/>
```
Comment (willfurnass) [inline site/pages/community/seminars.md]: Link to bio?

File: site/pages/community/seminars.md [hunk 4]
```diff
@@ -63,10 +62,10 @@ A supportive link:
 <br/><br/>
 
 ### April 16, 2018 &mdash; 15:30-16:30, Hicks building, lecture theatre 6
-<strong>David Hubber, Researcher at Ludwig-Maximilians-Universität Munich </strong><br/><br/>
+<strong>David Hubber, Researcher at Ludwig-Maximilians-Universität Munich<a href="/seminar_files/bio/david_hubber.md" target="popup" onclick="window.open('/seminar_files/bio/david_hubber.md','name','width=600,height=400')"> (Bio)</a></strong><br/><br/>
```
Comment (willfurnass) [inline site/pages/community/seminars.md]: Pop-up doesn't work.  Simpler/cleaner to just include the bio below the abstract?

File: site/pages/community/seminars.md [hunk 5]
```diff
@@ -13,18 +13,17 @@ If you would like to recommend a speaker or would like to give a talk then pleas
...
 <br/>
-**Abstract:** [[more info](https://www.sheffield.ac.uk/physics/news/programming-skills-sessions)]" 
-
+**Abstract:** The world’s digital data are still doubling in number every year or so, and research is a big culprit. Whether you measure it by volume, velocity or variety, research data is getting Big. All is not lost, though. It is challenging, and it does take a slightly different approach, but dealing with oodles of research data is not as terrifying as it seems. In this talk I aim to give researchers some practical ideas on working with “big” research data, illustrated with examples of how we manage stuff like this at EPCC. 
 <br/>
 
 # Previous Seminars #
 ### June 27, 2018 &mdash; 15:30-16:30, Hicks building, lecture theatre 7
-<strong>Joz Martin (Mathworks)</strong><br/><br/>
+<strong>Joz Martin (Mathworks) <a href="/seminar_files/bio/joz_martin.md" target="popup" onclick="window.open('/seminar_files/bio/joz_martin.md','name','width=600,height=400')">(Bio)</a></strong><br/><br/>
```
Comment (willfurnass) [inline site/pages/community/seminars.md]: Pop-up doesn't work.  Simpler/cleaner to just include the bio below the abstract?

**Review by willfurnass — CHANGES_REQUESTED** [review]
See comments in-line

File: site/pages/community/seminars.md [hunk 6]
```diff
@@ -11,24 +11,28 @@ If you would like to recommend a speaker or would like to give a talk then pleas
 # Upcoming Senimars #
 
 
-### September 12, 2018 &mdash; 15:30-16:30, Hicks building, lecture theatre 7
+### September 12, 2018 &mdash; 15:30-16:30, Hicks building, Lecture theatre 7
```
Comment (willfurnass) [inline site/pages/community/seminars.md]: Probably want to capitalise 'Theatre' too (here and elsewhere)

File: site/files/seminar_files/bio/david_hubber.md (hunk 1 above)
Comment (mozhgan-kch) [reply site/files/seminar_files/bio/david_hubber.md:1]: info is not available

File: site/files/seminar_files/bio/joz_martin.md (hunk 2 above)
Comment (mozhgan-kch) [reply site/files/seminar_files/bio/joz_martin.md:1]: info is not available

File: site/pages/community/seminars.md (hunk 6 above)
Comment (mozhgan-kch) [reply site/pages/community/seminars.md]: Done

File: site/pages/community/seminars.md [hunk 7]
```diff
@@ -11,25 +11,28 @@ If you would like to recommend a speaker or would like to give a talk then pleas
...
 
 <br/>
 
+**Bio:** Jos Martin is the senior engineering manager for parallel computing products at MathWorks. He has responsibility for all parts of Parallel Computing Toolbox, including the use of GPU and big data types in other areas. In addition, he is also responsible for MATLAB Drive, MATLAB Connector, and products that help link the desktop to the cloud. Before moving to a development role within MathWorks, he worked as a consultant in the U.K., writing large-scale MATLAB applications, particularly in the finance and automotive sectors. Prior to joining MathWorks in 2000, he held a Royal Society Post-Doctoral Fellowship at the University of Otago, New Zealand. His area of research was Experimental Bose-Einstein Condensation (BEC), a branch of low-temperature atomic physics.
+<br/><br/>
+<hr/>
+
 ### June 19, 2018 &mdash; 12:00-13:00, COM-G12-Main Lewin ###
-<strong>Dr Patricio Ortiz, Newcastle University <a href="/seminar_files/bio/patricio_ortiz.md" target="popup" onclick="window.open('/seminar_files/bio/patricio_ortiz.md','name','width=600,height=400')">(Bio)</a></strong><br/><br/>
+<strong>Dr Patricio Ortiz, Newcastle University</strong><br/><br/>
```
Comment (willfurnass) [inline site/pages/community/seminars.md]: Sheffield University

Comment (willfurnass) [inline site/pages/community/seminars.md]: Author name + affilliation not consistently formatted throughout page.

File: site/files/seminar_files/bio/david_hubber.md (hunk 1 above)
Comment (willfurnass) [reply site/files/seminar_files/bio/david_hubber.md:1]: File no longer needed?

File: site/files/seminar_files/bio/joz_martin.md (hunk 2 above)
Comment (willfurnass) [reply site/files/seminar_files/bio/joz_martin.md:1]: File no longer needed?

File: site/pages/community/seminars.md [hunk 8]
```diff
@@ -11,25 +11,28 @@ If you would like to recommend a speaker or would like to give a talk then pleas
 # Upcoming Senimars #
 
 
-### September 12, 2018 &mdash; 15:30-16:30, Hicks building, lecture theatre 7
+### September 12, 2018 &mdash; 15:30-16:30, Hicks building, Lecture Theatre 7
 <strong>Rob Baxter (EPCC)</strong><br/><br/>
```
Comment (willfurnass) [inline site/pages/community/seminars.md]: No bio

**Review by willfurnass — CHANGES_REQUESTED** [review]
See new in-line comments.

File: site/files/seminar_files/bio/joz_martin.md (hunk 2 above)
Comment (mozhgan-kch) [reply site/files/seminar_files/bio/joz_martin.md:1]: No

File: site/files/seminar_files/bio/david_hubber.md (hunk 1 above)
Comment (mozhgan-kch) [reply site/files/seminar_files/bio/david_hubber.md:1]: no

**Pull request merged**